control_plane/input_params/base_vars.yml

```diff
@@ -133,8 +129,8 @@ host_mapping_file_path: ""
 
 # The nic/ethernet card that needs to be connected to configure infiniband switch
 # This nic will be configured by Omnia for the DHCP server.
-# Default value of nic is ib0
-ib_network_nic: "ib0"
+# Default value of nic is ib1
```

Comment:
why do we assume there are 2 infiniband connections?

Reply:
We don't need 2 infiniband connections. 
Earlier we had kept ib0 as default value for IB nic. But ib1 works for our development setup. So, we have changed the default values to those working for our setup. 
User can change these values according to their need.

Reply:
The default values should match the hardware architecture we define in [Example System Design](https://github.com/dellhpc/omnia/blob/devel/docs/PREINSTALL_OMNIA_APPLIANCE.md). It is not safe to assume the default user will have multiple infiniband connections.

Reply:
Sure @j0hnL. I'll change it back to ib0.